src/meta/src/manager/catalog/fragment.rs

```diff
@@ -1079,6 +1079,11 @@ impl FragmentManager {
                 for actor in &mut fragment.actors {
                     if let Some(node) = actor.nodes.as_mut() {
                         visit_stream_node(node, |node_body| match node_body {
+                            // rate limit for cdc backfill
+                            NodeBody::StreamCdcScan(ref mut node) => {
```

Comment:
This part looks correct, but the fragment_mask in L1075 above seems to be not set for `StreamCdcScan`.

Reply:
I found it in the comment. 
> /// Include StreamScan and StreamCdcScan
    StreamScan = 16,

Reply:
Seems like it should be separate. We can fix it in a separate PR. They are fundamentally different kinds of scans.